Sources/SwiftUINavigation/NavigationDestination.swift

```diff
@@ -91,4 +98,18 @@
         .bind(self.$isPresented, to: self.$isPresentedState)
     }
   }
+
+  #if os(iOS) || os(tvOS)
```

Comment:
Why not just `if @available`?

Reply:
Availability unfortunately introduces performance implications in `ViewBuilder`, where newer versions are wrapped in `AnyView`. Luckily, availability really isn't even the core issue here, since we don't need to hide away APIs that are unavailable. We simply have an OS cut-off that we want to selectively apply a runtime fix to.

Reply:
This seems equivalent to

```swift
if #available(iOS 16.4, macOS 13.3, tvOS 16.4, watchOS 9.4) {
  return false 
} else {
  return true 
}
```

Reply:
Oh, because my brain didn't quite make it to that refactor 😅

Pushing a simplification, thanks!